env0/utils.go

```diff
@@ -19,6 +19,10 @@ type CustomResourceDataField interface {
 	WriteResourceData(fieldName string, d *schema.ResourceData) error
 }
 
+type ResourceDataSliceStructValueWriter interface {
```

Comment:
This interface is for custom writing.
see `func (c *ConfigurationVariableSchema) ResourceDataSliceStructValueWrite(values map[string]interface{}) error {`

Unit test is below.